.github/workflows/build-presets.yml

```diff
@@ -66,3 +66,30 @@ jobs:
         ./install_requirements.sh > /dev/null
         cmake --preset ${{ matrix.preset }}
         cmake --build cmake-out --parallel
+
+  windows:
+    uses: pytorch/test-infra/.github/workflows/windows_job.yml@main
+    strategy:
+      fail-fast: false
+      matrix:
+        preset: [pybind]
+    with:
+      job-name: build
+      ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
+      submodules: recursive
+      timeout: 90
+      script: |
+        set -eux
+        conda init powershell
+        powershell -Command "& {
```

Comment:
very nice! So it still doesn't work via bash? If so, very curious why powershell specifically is needed...

Reply:
Yeah there are some compiler errors when using bash. I think this is good for now, we can look into the bash issue later